examples/hello-world/ml-to-fl/jobs/lightning_client_api/app/custom/autoencoder.py

```diff
@@ -193,24 +193,31 @@ def predict_dataloader(self):
 
 
 def cli_main():
-    # (1) patch the LightningModule
-    flare.patch(LitAutoEncoder)
+
     cli = LightningCLI(
         LitAutoEncoder,
         MyDataModule,
         seed_everything_default=1234,
         run=False,  # used to de-activate automatic fitting.
-        trainer_defaults={"callbacks": ImageSampler(), "max_epochs": 1},
+        trainer_defaults={
+            "callbacks": [ImageSampler()],
+            "max_epochs": 1,
+        },
         save_config_kwargs={"overwrite": True},
     )
+    # (1) patch the lightning trainer
+    flare.patch(cli.trainer)
+
+    print("--- test global model ---")
```

Comment:
Let's add a comment here like `# (2) evaluate the current global model to allow server-side model selection"